TsRandomizer/LevelObjects/Other/HistoricalDocuments.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Timespinner.GameObjects.BaseClasses;
+using TsRandomizer.IntermediateObjects;
+
+namespace TsRandomizer.LevelObjects.Other
+{
+	[TimeSpinnerType("Timespinner.GameObjects.Events.EnvironmentPrefabs.L11_Lab.EnvPrefabLabHistoricalDocuments")]
+	// ReSharper disable once UnusedMember.Global
+	class HistoricalDocuments: LevelObject
+	{
+		public HistoricalDocuments(Mobile typedObject) : base(typedObject)
+		{
+			TimeSpinnerGame.Localizer.OverrideKey("q_ram_4_lun_29alt",
```

Comment:
Should probably check seed options here to see if the string needs to be overridden

Reply:
Added